djangocms_link/models.py

```diff
@@ -60,6 +63,21 @@ class AbstractLink(CMSPlugin):
     target = models.CharField(_('target'), blank=True, max_length=100,
                               choices=TARGET_CHOICES)
 
+    # These attributes are already managed by other fields
+    EXCLUDED_KEYS = [
+        'href',
+        'target',
+    ]
+
+    attributes = AttributesField(
+        _('link tag attributes'), excluded_keys=EXCLUDED_KEYS,
+        help_text=_('Optional. Link HTML tag attributes'),
+    )
+
+    @property
+    def attributes_str(self):
+        return AttributesField.to_str(self, 'attributes')
```

Comment:
actually ``AttributesField`` could use ``.contribute_to_class`` to automatically add an attribute to the model for this. e.g ``<field_name>_str``.
Like djangos built in choice field does for ``get_<fieldname>_display``: https://github.com/django/django/blob/master/django/db/models/fields/__init__.py#L703-L704